source/plugins/go/src/oms.go

```diff
@@ -87,9 +87,9 @@ const WindowsContainerLogPluginConfFilePath = "/etc/amalogswindows/out_oms.conf"
 const IPName = "ContainerInsights"
 
 const defaultContainerInventoryRefreshInterval = 60
-
 const kubeMonAgentConfigEventFlushInterval = 60
 const defaultIngestionAuthTokenRefreshIntervalSeconds = 3600
+const containerLogV2PollingIntervalMinutes = 3
```

Comment:
why 3 minutes?

Reply:
there is a 2 minutes wait in fetchContainerLogV2FromDCR function so the total time inteval is 5 mins